### ISSUE TITLE
Unable to clean directory with spaces

### ISSUE THREAD

**Issue by adamoxley** [vapor/toolbox#286]
Attempting to clean a project directory with `vapor-beta` fails with the following;

```
:~/Proj V4 $ vapor-beta clean                                                                                                                                                                                                                                                           
error:
ls: V4: No such file or directory
```

Moving the directory to a non-spaced name works as expected with `vapor-beta`;

```
:~/Proj V4 $ cd ../
:~/Proj V4 $ mv Proj\ V4 Proj-V4
:~/Proj V4 $ cd Proj-V4
:~/Proj-V4 $ vapor-beta clean

  DerivedData        cleaned file
  xcodeproj          cleaned file
  .build             cleaned file
  Package.resolved   use [--update,-u] flag to remove this file during clean
```

## Steps to reproduce

Follow the above command line input on a spaced directory

## Expected behavior

Work the same as `vapor clean` on a directory with a space as per;

```
:~/Proj V4 $ vapor-beta clean
error:
ls: V4: No such file or directory

:~/Proj V4 $ vapor clean -y
Cleaning [Done]
```

## Actual behavior

Attempts to `ls` to directory named characters after the space (e.g V4 in this case)

## Environment

```
$ brew info vapor/tap/vapor-beta
vapor/tap/vapor-beta: stable 18.0.0-beta.19, HEAD
```

Vapor Framework version: 4
OS: macOS Catalina 10.15

**Comment by adamoxley** [comment]
Likely related issue https://github.com/vapor/toolbox/issues/280

**Comment by gwynne** [comment]
This should now be fixed with the `18.0.0-beta.29` release (and the subsequent GM `18.0.0` release).
